tests/wicked/before_test.pm

```diff
@@ -138,6 +138,13 @@ sub run {
             zypper_ar($repo_url . generate_version('_') . '/', name => 'wicked_maintainers', no_gpg_check => 1, priority => 60);
             $package_list .= ' ndisc6';
         }
+        if (check_var('WICKED', 'startandstop')) {
+            # No firewalld on sles 12-SP5 (bsc#1180116)
+            if (!is_sle('<=12-SP5')) {
```

Comment:
```suggestion
            if (is_sle('>12-SP5')) {
```
it is exactly the same but it is always easier to read and understand true expressions ( have read this somewhere :) )

Reply:
ack